packages/wonder-blocks-layout/docs.md

```diff
@@ -91,3 +98,136 @@ const styleSheets = {
     }}
 </MediaLayout>
 ```
+
+#### 2. Defining shared styles for all sizes
+
+You can use the `all` key to define styles for all the different sizes. This
+means that by using this key, all the sizes (small, medium, large) will use the
+styles defined in `all`, and in case there are duplicate properties, the
+specific sizes will take more importance.
```

Comment:
I think I wrote tests for tests case but obviously not examples. Thanks for fleshing out this part of the docs. 

Nit: `the specific sizes` => `more specific sizes` since we also support `mdOrSmaller` and `mdOrLarger`.  I think it'd be good to show one of those in action as well.